examples/va/nfv/sriov/edpm/nova_sriov.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# CHANGEME:
+#
+# - Change cpu-pinning-nova ConfigMap's "data['25-cpu-pinning-nova.conf']" to use CPU sets
+#   proper for your environment
+# - Change sriov-nova ConfigMap's "data['03-sriov-nova.conf']" "devname" and "physical_network"
+#   parameters to the proper SRIOV interface and network for your environment
+#
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cpu-pinning-nova
+data:
+  25-cpu-pinning-nova.conf: |
+    [compute]
+    cpu_shared_set = 0-3,24-27 # CHANGEME
+    cpu_dedicated_set = 8-23,32-47 # CHANGEME
+    reserved_host_memory_mb = 4096
```

Comment:
this is not the corect way to reserve host memory on a numa host

and since you have enabled CPU pinning you also need the numatoplogy filter or the scheduler will select host that cant actually fit the VM
due to memory/sriov device affintiy